### PR TITLE
removed redundant if statement

### PR DESCRIPTION
**Pull request by oguzhancevik**
Removed redundant if statement. 
This commit helps to open source community for Hacktoberfest.

### CODE REVIEW

File: core/src/main/java/jenkins/security/BasicHeaderProcessor.java [hunk 1]
```diff
@@ -125,11 +125,7 @@ protected boolean authenticationIsRequired(String username) {
         // BASIC protocol is desirable. This behaviour is also consistent with that provided by form and digest,
         // both of which force re-authentication if the respective header is detected (and in doing so replace
         // any existing AnonymousAuthenticationToken). See SEC-610.
-        if (existingAuth instanceof AnonymousAuthenticationToken) {
-            return true;
-        }
-
-        return false;
+        return existingAuth instanceof AnonymousAuthenticationToken;
```
Comment (daniel-beck) [inline core/src/main/java/jenkins/security/BasicHeaderProcessor.java:128]: This change makes the preceding comment much more difficult to understand.

File: core/src/main/java/jenkins/management/AdministrativeMonitorsDecorator.java [hunk 2]
```diff
@@ -123,10 +123,6 @@ public boolean shouldDisplay() {
             }
         }
 
-        if (getActiveAdministrativeMonitorsCount() == 0) {
-            return false;
-        }
-
-        return true;
+        return getActiveAdministrativeMonitorsCount() != 0;
```
Comment (daniel-beck) [inline core/src/main/java/jenkins/management/AdministrativeMonitorsDecorator.java:126]:
```suggestion
        return getActiveAdministrativeMonitorsCount() > 0;
```

**Review by daniel-beck — COMMENTED** [review]
How do I configure the IntelliJ debugger to break after the method call in the last line is evaluated, but before I leave the method?

(I prefer the more verbose if statements because I can easily break on `return` statement lines, and it's trivial to see what is being returned; never figured out how to easily accomplish the equivalent when it's all on a single line.)

File: core/src/main/java/jenkins/security/BasicHeaderProcessor.java (hunk 1 above)
Comment (oguzhancevik) [reply core/src/main/java/jenkins/security/BasicHeaderProcessor.java:128]: Hi, I suggest you can use (Ctrl+I). This shortcut evaluates any code statement. We must care about code readability.

Comment (daniel-beck) [reply core/src/main/java/jenkins/security/BasicHeaderProcessor.java:128]: I'm confused by this comment.

Comment (oguzhancevik) [reply core/src/main/java/jenkins/security/BasicHeaderProcessor.java:128]: This statement says if existingAuth object is derived AnonymousAuthenticationToken return true, else return false. I removed only if statement.

Comment (daniel-beck) [reply core/src/main/java/jenkins/security/BasicHeaderProcessor.java:128]: I know what you did. I'm confused by your comment 

> Hi, I suggest you can use (Ctrl+I). This shortcut evaluates any code statement. We must care about code readability.

in response to me pointing out that the preceding source code comment makes no sense after this change.

**Comment by oguzhancevik** [comment]
Thanks @oleg-nenashev

**Comment by oleg-nenashev** [comment]
@oguzhancevik just to clarify, why did you close the pull request? In our org we consider only genuine contributions when it comes to Hacktoberfest or any other events. It is not enough to submit something viable and then close the pull request

**Comment by oguzhancevik** [comment]
Hi @oleg-nenashev Sorry, there was a mistake. I opened it again.

File: core/src/main/java/jenkins/management/AdministrativeMonitorsDecorator.java [hunk 3]
```diff
@@ -123,10 +123,6 @@ public boolean shouldDisplay() {
             }
         }
 
-        if (getActiveAdministrativeMonitorsCount() == 0) {
-            return false;
-        }
-
-        return true;
+        return getActiveAdministrativeMonitorsCount() > 0;
```
Comment (MRamonLeon) [inline core/src/main/java/jenkins/management/AdministrativeMonitorsDecorator.java:126]: To be more precise:
```suggestion
        return getActiveAdministrativeMonitorsCount() != 0;
```

**Review by rsandell — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]
The changes have sense to me. Only a minor suggestion

**Review by oleg-nenashev — APPROVED** [review]
All refactorings here are simple enough and improve code readability. +1 for going ahead with the merges.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by daniel-beck** [comment]
I would still like to get a response to my question about how to properly break and see the return value. This is at minimum a waste of time, perhaps even a loss of functionality, while shortening the code an irrelevant amount.

**Comment by oguzhancevik** [comment]
> I would still like to get a response to my question about how to properly break and see the return value. This is at minimum an added waste of time when stepping through the program, perhaps even a loss of functionality, while shortening the code an irrelevant amount.

You can see the return value with any IDE. I'm using IntelliJ. I can easily debug this code and I can see the return value.

**Comment by timja** [comment]
> I would still like to get a response to my question about how to properly break and see the return value. This is at minimum an added waste of time when stepping through the program, perhaps even a loss of functionality, while shortening the code an irrelevant amount.

you can add a breakpoint and 'Evaluate expression', a little bit more time but not too bad, code looks simpler to me

**Pull request merged**